submissions/smillims/js-dom/css/style.css

```diff
@@ -0,0 +1,179 @@
+@charset "UTF-8";
+
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+:focus,
+:active {
+  outline: none;
+}
+
+a:focus,
+a:active {
+  outline: none;
```

Comment:
This is problematic, because it makes the site inaccessible to users who use the keyboard for navigation (they would not see any visual indication of moving from one link to the next one). It is ok to unset default styles if they do not match the design, but you need to provide alternative styles to indicate the active menu item. Take a look at `:active` and `:focus` pseudo classes